main.mk

```diff
@@ -41,6 +41,14 @@ endif
 %.txt: %.xml
 	$(xml2rfc) $< -o $@ --text
 
+%.html: %.jrxml rfc2629xslt/rfc2629.xslt
```

Comment:
It's currently in the top-level make file (the one that includes this) -- I guess I should move it down?